### PR TITLE
Changelog for 0.18

### PR DESCRIPTION
**Pull request by rjrjr**

### CODE REVIEW

**Comment by pyricau** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Change Log
 ==========
 
+Version 0.18 *(2015-07-14)*
+------------------
+ * Destroying a scope recursively destroys its children first, like it used to.
+   (0.17 API quake incorrectly reversed this.)
+
+ * Now crashes (fail fast!) when doing service lookup in a dead scope.
```
Comment (loganj) [inline CHANGELOG.md]: "throws".  Crashing is up to you.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
 ==========
 
+Version 0.18 *(2015-07-14)*
+------------------
+ * Destroying a scope recursively destroys its children first, like it used to.
+   (0.17 API quake incorrectly reversed this.)
+
+ * Now crashes (fail fast!) when doing service lookup in a dead scope. 
+
+ * Manually fall back to app context when service not found. Works around
```
Comment (loganj) [inline CHANGELOG.md]: "falls back"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+   window in activity context life cycle where they're not quite attached yet.
```
Comment (loganj) [inline CHANGELOG.md]: "Works around window in activity context" reads like there are some words missing.  Also "they" doesn't really agree with its antecedent.

**Comment by loganj** [comment]
English nits, LGTM

**Pull request merged**